src/components/NavBar/index.jsx

```diff
@@ -0,0 +1,14 @@
+const NavBar = ({ status }) => {
+  return (
+    <div
+      className={`border text-white h-screen bg-cyan-600 md:p-4 p-8 sm:w-1/12 w-1/2 sm:static absolute z-10 ${status} sm:block sm:min-w-48`}
```

Comment:
when lines get really long like this we tend to format them as a class per line, ie:
```
    <div className={`
      border
      text-white
      h-screen
      ...etc
    `}
```